pytest_fixtures.py

```diff
@@ -0,0 +1,90 @@
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--test_db_url',
```

Comment:
Yes, this way you would get the same result in for that case and it's shorter.
I guess it looks more consistent and simpler to pass the URI in one single pytest command option and I can still get individual Makefile variables as I do now.